CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/report/compare/release-3.8.0...master
 
+### Fixed
+- Remove unsecured call such as !!.
```

Comment:
It's less about security and more about diagnosability

Reply:
I will change that for:
Add meaningful information during exception throwing.
Is that better?

Reply:
```suggestion
- Emit useful errors from `RelativePerformanceStabilityJudge`.
```